cvmfs/loader.cc

```diff
@@ -736,6 +749,21 @@ int FuseMain(int argc, char *argv[]) {
     fuse_opt_add_arg(mount_options, "-osuid");
     LogCvmfs(kLogCvmfs, kLogStdout, "CernVM-FS: running with suid support");
   }
+
+ if( options_manager->GetValue("CVMFS_CPU_AFFINITY", &parameter) ) {
+     cpu_set_t mask;
+     vector<int> cpus = splitToInts( parameter, ',' );
+     CPU_ZERO(&mask);
+     for( int i : cpus ) {
+        CPU_SET( i, &mask );
+     }
+
+     LogCvmfs(kLogCvmfs, kLogStdout, "Setting CPU Affinity to %s", parameter.c_str() );
+     int err = sched_setaffinity( 0, sizeof(mask), &mask );
```

Comment:
We'll need to ignore the request on macOS or use Mac's native API; I don't think `sched_setaffinity` exists there.